dashboard/package.json

```diff
@@ -6,10 +6,9 @@
   "homepage": "./",
   "scripts": {
     "build": "npm-run-all build-css compile-lang build-js",
-    "build-css": "sass src/ --no-source-map && yarn run copy-clr",
+    "build-css": "sass src/ --no-source-map && yarn run",
     "build-js": "export NODE_OPTIONS=--max-old-space-size=4096 && craco build",
     "compile-lang": "formatjs compile-folder lang src/locales/ --ast --format simple",
-    "copy-clr": "shx cp ./node_modules/@clr/ui/legacy-clr-ui-dark.min.css public/clr-ui-dark.min.css && shx cp node_modules/@clr/ui/legacy-clr-ui.min.css public/clr-ui.min.css && shx cp ./node_modules/@clr/ui/legacy-clr-ui-dark.min.css.map public/clr-ui-dark.min.css.map && shx cp ./node_modules/@clr/ui/legacy-clr-ui.min.css.map public/clr-ui.min.css.map",
```

Comment:
This hack is no longer required! The light/dark theme switching works out of the box now :)